packages/manager/apps/web/client/app/domain/webhosting/order/steps/payment/domain-webhosting-order-steps-payment.html

```diff
@@ -0,0 +1,35 @@
+<oui-step-form
+    data-header="{{:: 'domain_webhosting_order_payment_step_title' | translate }}"
+    data-on-focus="$ctrl.getCheckout()"
+    data-loading="$ctrl.stepper.loadingCheckout"
+    data-valid="$ctrl.autoPayWithPreferredPaymentMethod != null && $ctrl.agreeContracts"
+    data-prevent-next="true"
+    data-on-cancel="$ctrl.goBackToDashboard()"
+    data-on-submit="$ctrl.preparePayment()">
+
+    <div class="row">
+        <dl class="oui-description oui-description_horizontal oui-heading_underline col-md-2">
+            <dt data-translate="domain_webhosting_order_payment_total_price_without_tax"></dt>
+            <dd data-ng-bind=":: $ctrl.prices.withoutTax.text"></dd>
+            <dt data-translate="domain_webhosting_order_payment_total_price_with_tax"></dt>
+            <dd data-ng-bind=":: $ctrl.prices.withTax.text"></dd>
+        </dl>
+    </div>
+
+    <form
+        name="$ctrl.paymentMeans"
+        novalidate>
```

Comment:
Both component name `order-payment-means` and `order-contracts` are too generic in my opinion.